Add clarification on judges forgetting to put a sight blocker (#1052)
thewca/wca-regulations: wca-guidelines.md

- B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.
- B1b+) [RECOMMENDATION] Blindfolds should be checked by the WCA Delegate before use in the competition.
- B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must (ask and) receive confirmation from the judge each time before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without confirmation from the judge: disqualification of the attempt (DNF).

> - B4c+) [ADDITION] An attempt should not be disqualified solely due to the judge not putting a sight blocker. However, the WCA Delegate may replace the attempt with an extra attempt if they have a reason to suspect that the competitor gained an unfair advantage.

Reviewer: I think this needs to make explicit that this does *not* apply if https://www.worldcubeassociation.org/regulations/#B4c3 is in effect (in which case the judge was not expected to use a sight blocker, and the competitor is responsible).

Author: I actually thought about it, but I can't figure out how to clarify it correctly. Maybe an exception is needed here, like:

```
... the competitor gained an unfair advantage. Exception: if the competitor had to put a sight blocker themselves (see [Regulation B4c3](regulations:regulation:B4c3)), the attempt should be disqualified, at the discretion of the WCA Delegate.
````

And I still don't like this wording.

Reviewer 2: > thus making it look to have the same level of importance.

What would be the problem with that?

Author: > What would be the problem with that?

A competition with all scramblers signatures missing won't probably cause a debate on whether the results should stay or not, but I don't think it would be fine to have the same for the sight blockers.

Reviewer 2: Understood. What about changing "should not be disqualified" to "May remain valid" (or something like that)? With that wording, it would be a possibility, instead of a recommendation.

Author: It would sound a way better. But maybe it should even be reworded to make it an exception? To clearly define that it's not a normal case and should be trated seriously.

> - B4c+) [ADDITION] An attempt should not be disqualified or replaced with an extra attempt solely due to the judge not putting a sight blocker. However, the WCA Delegate may replace the attempt with an extra attempt if they have a reason to suspect that the competitor gained an unfair advantage. Exception: if the competitor had to put a sight blocker themselves (see [Regulation B4c3](regulations:regulation:B4c3)), the attempt should be disqualified, at the discretion of the WCA Delegate.

Reviewer 3: ```suggestion
- B4c+) [ADDITION] if the judge forgot to put a sight blocker (or did not put it immediately), the WCA Delegate should always grant an extra attempt.
- B4c++) [ADDITION] If the WCA Delegate does not suspect any infraction of the regulations by the competitor, the original attempt may stand.
```

Reviewer 3: @lgarron can you think of alternate wording here? i don't enjoy the wording of this too much, but i think the idea is there

Reviewer 4: "... does not suspect any infraction of the regulations..." might sound less clunky

- B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.
- B1b+) [RECOMMENDATION] Blindfolds should be checked by the WCA Delegate before use in the competition.
- B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must (ask and) receive confirmation from the judge each time before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without confirmation from the judge: disqualification of the attempt (DNF).

> - B4c+) [ADDITION] If the judge forgot to put a sight blocker (or did not put it immediately), the WCA Delegate should always grant an extra attempt.

Reviewer 3: ```suggestion
- B4c+) [ADDITION] If the judge forgot to put a sight blocker, or places it in a way that the competitor's view of the puzzle is not fully blocked, the WCA Delegate should grant an extra attempt.
```

Reviewer 3: changed must -> should to account for the fact that it's not a very good idea to give an extra if the competitor DNFs anyways

Reviewer 5: I much prefer this wording. The "immediately" doesn't really matter except for top 3bld solvers, and the "always" doesn't sit well with me.

> - B4c++) [ADDITION] If the WCA Delegate does not suspect any infraction of the regulations by the competitor, the original attempt may stand.

Reviewer: Nit: I'd avoid "infraction"  for international audiences, in favor of something more direct:
```suggestion
- B4c++) [ADDITION] If the WCA Delegate does not suspect that the competitor intentionally violated any WCA Regulations, the original attempt may stand.
```

Author: Shouldn't `any Regulations` here be downcase, i.e. `any regulations`?

Reviewer: Either is technically fine (sort of how a Delegate is also a delegate), I tend to uppercase.

Author: But for the delegate we always use the uppercase, while for multiple/undefined regulations downcase (e.g. [5b5](https://www.worldcubeassociation.org/regulations/#5b5)).

Reviewer 3: Updated comment to say WCA Regulations here. (I have seen capital R be preceded by WCA more often than not)
